app/academics/undergraduate/scholarship/[id]/page.tsx

```diff
@@ -0,0 +1,18 @@
+import { getScholarship } from '@/apis/academicsServer';
+
+import HTMLViewer from '@/components/editor/HTMLViewer';
+import PageLayout from '@/components/layout/pageLayout/PageLayout';
+
+export default async function UndergraduateGuidePage({ params }: { params: { id: number } }) {
+  const { name, description } = await getScholarship(params.id);
+
+  return (
+    <PageLayout
+      title={name.length > 20 ? name.replace(/\([^)]*\)/g, '') : name}
```

Comment:
title 문자열 수정하는게 흔한 일은 아니니 주석 달면 좋을 것 같아요!

```suggestion
// 타이틀이 긴 경우 정규표현식으로 괄호 내부 내용을 제거
// ex) 교외장학금 (현송문화재단, 유한재단, ...) -> 교외장학금
    <PageLayout
      title={name.length > 20 ? name.replace(/\([^)]*\)/g, '') : name}
```